### ISSUE TITLE
TypeError: Cannot read property 'NO_CONTENT' of undefined (HttpStatus issue)

### ISSUE THREAD

**Issue by HlibDerbenov** [nestjs/swagger#1379]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

We want to use NestJS v.8x functionality for API versioning so we updated _@nestjs/core_ and _@nestjs/common_ (**7.4.4 => 8.0.0-alpha.5** and **7.6.13 => 8.0.0-alpha.5** respectively). And we use Swagger to generate docs for API (_@nestjs/swagger_ - **4.8.0**). After the update of 2 first packages listed above we have the following error:

```
TypeError: Cannot read property 'NO_CONTENT' of undefined
    at Object.ApiNoContentResponse (projectFolder\node_modules\@nestjs\swagger\dist\decorators\api-response.decorator.js:35:142)
```

You can easily change _ApiNoContentResponse_ to any other Swagger decorator like _ApiForbiddenResponse_, same behaviour occurs.

I guess it happens because @nestjs/common/enums/http-status.enum file was incorrectly modified while _@nestjs/common_ development. Please have a look at it.

**@nestjs/common@7.4.4:**
```
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.HttpStatus = void 0;
var HttpStatus;
(function (HttpStatus) {
    HttpStatus[HttpStatus["CONTINUE"] = 100] = "CONTINUE";
    /* other codes - not important */
    HttpStatus[HttpStatus["NO_CONTENT"] = 204] = "NO_CONTENT";
    /* other codes - not important */
    HttpStatus[HttpStatus["HTTP_VERSION_NOT_SUPPORTED"] = 505] = "HTTP_VERSION_NOT_SUPPORTED";
})(HttpStatus = exports.HttpStatus || (exports.HttpStatus = {}));
```

**@nestjs/common@8.0.0-alpha.5:**

```
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
```

Probably that's why when this file is required: `const http_status_enum_1 = require("@nestjs/common/enums/http-status.enum");` it returns `{}` and not `{ HttpStatus: {...}}` or something like it.

## Expected behavior

No error occurs.

## Minimal reproduction of the problem with instructions

```
import { Controller, Get } from '@nestjs/common';
import { ApiNoContentResponse } from '@nestjs/swagger';

@Controller('test')
export class TestController {
  constructor() {}

  @Get()
  @ApiNoContentResponse({ description: 'Test' })
  test() {
    return 0;
  }
}
```

## What is the motivation / use case for changing the behavior?

We cannot use Swagger with NestJS v.8x.

## Environment

<pre><code>
Nest version: 7.x.x

 
For Tooling issues:
- Node version: 14.15.x  
- Platform:  Windows

Others:
- @nestjs/core: 8.0.0-alpha.5
- @nestjs/common: 8.0.0-alpha.5
- @nestjs/swagger: 4.8.0
</code></pre>

**Comment by HlibDerbenov** [comment]
@kamilmysliwiec we also discovered something:

1) We just added the simplest export `module.exports = { HttpStatus: { NO_CONTENT: 204 } }` to http-status-enum.js just to test if not exported `HttpStatus` is a root cause — issue is now gone, project is succesfully launched with no errors; meaning that this is a _@nestjs/common_ package issue;
2) We have global prefix 'v1' so that all our routes start with /v1. We wanted to add versioning only for 1 endpoint so `@Version('2')` decorator was attached to specific method. But instead we received /v1/v2/... endpoint. So we need to remove global prefix and modify all controllers like `@Controller({ path: 'somePath' version: '1' })`, right?
3) After we made changes according to point 2 (above) endpoint /v2/... is succesfully mapped, but Swagger UI shows endpoint without prefix - will it be fixed soon? So that swager takes into account the `version` in the controller.

Thanks in advance.
